### ISSUE TITLE
Test updated JSON Schema with at-risk items removed

### ISSUE THREAD

**Issue by mmccool** [w3c/wot-thing-description#858]
I updated the JSON Schema for PR https://github.com/w3c/wot-thing-description/pull/856 to remove at-risk items, but it needs to be tested (eg to make sure it is a valid JSON schema, and correctly accepts all valid TDs in our test suite).

**Comment by egekorkan** [comment]
The schema has changed only to remove some security schemes and once the content of an enum, so it should not break anything but testing now

**Comment by egekorkan** [comment]
Warning: There was a mismatch between the [td-schema used by playground](https://github.com/thingweb/thingweb-playground/blob/master/WebContent/td-schema.json) and the one in this repo. Nothing too important but still.
Some things were not shown in the GitHub rendered version and here are the issues:
- You seem to have reordered the definitions, which is not good since we want a definition to not have any reference to an undefined definition, i.e. if we didn't define title yet, we should not use it in another definition. This was an issue some 
- Security is not able to accept a string anymore which is required by the TD spec, it used to be: 
```json
        "security": {
            "oneOf": [
                {
                    "type": "array",
                    "items": {
                        "type": "string"
                    }
                },
                {
                    "type": "string"
                }
            ]
        },
```
- Scopes should be string or string of arrays and should be in a definition
- @context should always start with the td vocabulary if it is an array whereas now it is possible to start with any uri
- subprotocol is not using the definition

**Comment by egekorkan** [comment]
@mmccool could you briefly say what you needed to change. I think it is only for the security schemes?

**Comment by sebastiankb** [comment]
this issue was fixed by PR #863
